scripts/verify_sitemap_links.py

```
@@ -0,0 +1,302 @@
+    """
+    parsed = urlparse(url)
+    if parsed.scheme not in ("http", "https"):
+        return False, f"InvalidScheme:{parsed.scheme}"
+    if parsed.netloc not in ALLOWED_HOSTS:
+        return False, f"DisallowedHost:{parsed.netloc}"
+
+    req = urllib.request.Request(
+        url,
+        method="GET",
+        headers={"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64)"}
+    )
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)